extensions/wikia/WikiaApi/WikiaApiQueryAllUsers.php

```diff
@@ -345,36 +333,36 @@ private function local_users() {
 					$lastUserData['blockreason'] = $block_reason;
 				}
 				# edit count
-				if ($this->fld_editcount) {
-					$lastUserData['editcount'] = intval($row->user_editcount);
+				if ( $this->fld_editcount ) {
+					$lastUserData['editcount'] = intval( $row->user_editcount );
 				}
-				if ($this->fld_registration) {		
-					$user_registration = $this->getUserRegistration($row->user_id);
-					$lastUserData['registration'] = wfTimestamp(TS_ISO_8601, $user_registration);
+				if ( $this->fld_registration ) {
+					$user_registration = $this->getUserRegistration( $row->user_id );
+					$lastUserData['registration'] = wfTimestamp( TS_ISO_8601, $user_registration );
 				}
 			}
 
 			// Add user's group info
 			if ( $this->fld_groups && $row->cnt_groups > 0 ) {
 				$use_group = array();
-				$groups = explode(";", $row->all_groups);
-				if ( !empty($groups) ) {
+				$groups = explode( ";", $row->all_groups );
+				if ( !empty( $groups ) ) {
 					foreach ( $groups as $group ) {
 						if ( !in_array( $group, $use_group ) ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to not nest more than 4 "if", "for", "while", "switch" and "try" statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS134)